src/sinol_make/commands/inwer/__init__.py

```diff
@@ -0,0 +1,115 @@
+from sinol_make.commands.inwer import inwer_util
+
+
+class Command(BaseCommand):
+    """
+    Class for "inwer" command.
+    """
+
+    def get_name(self):
+        return "inwer"
+
+    def configure_subparser(self, subparser: argparse.ArgumentParser):
+        parser = subparser.add_parser(
+            self.get_name(),
+            help='Verify if input files are correct',
+            description='Verify if input files are correct'
+        )
+
+        parser.add_argument('inwer', type=str, nargs='?',
+                            help='path to inwer executable')
+        parser.add_argument('--tests', type=str, nargs='+',
+                            help='test to verify, for example in/abc{0,1}*')
+        parser.add_argument('--cpus', type=int,
+                            help=f'number of cpus to use, you have {mp.cpu_count()} available')
+        add_compilation_arguments(parser)
+
+    def compile_inwer(self, args: argparse.Namespace):
+        self.inwer_executable, compile_log = inwer_util.compile_inwer(self.inwer, args)
+        if self.inwer_executable is None:
+            print(util.error('Compilation failed.'))
+            os.system("head -c 500 %s" % compile_log)
+            exit(1)
+        else:
+            print(util.info('Compilation successful.'))
+
+    def verify_test(self, execution: InwerExecution) -> VerificationResult:
+        """
+        Veryfies test and returns results.
+        """
+        output_dir = os.path.join(os.getcwd(), 'cache', 'executions', execution.test_name)
+        os.makedirs(output_dir, exist_ok=True)
+        output_file = os.path.join(output_dir, 'inwer.out')
+
+        command = f'{execution.inwer_exe_path} {execution.test_name} < {execution.test_path} > {output_file}'
+        exit_code = os.system(command)
+
+        return VerificationResult(
+            execution.test_path,
+            exit_code == 0,
+            open(output_file, 'r').read()
+        )
+
+    def verify_and_print_table(self) -> dict[str, TestResult]:
+        results = {}
+        sorted_tests = sorted(self.tests, key=lambda x: x[0])
+        executions: list[InwerExecution] = []
+        for test in sorted_tests:
+            results[test] = TestResult(test)
+            executions.append(InwerExecution(test, results[test].test_name, self.inwer_executable))
+
+        table_data = TableData(results, 0)
+        print('Verifying tests...\n\n')
+        with mp.Pool(self.cpus) as pool:
+            for i, result in enumerate(pool.imap(self.verify_test, executions)):
+                table_data.results[result.test_path].update_result(result.valid, result.output)
+                inwer_util.print_view(table_data)
+
+        return results
+
+    def run(self, args: argparse.Namespace):
+        if not util.check_if_project():
+            print(util.warning('You are not in a project directory (couldn\'t find config.yml in current directory).'))
+            exit(1)
```

Comment:
This looks like an error, not a warning. Is there a reason to make it a warning?

Reply:
Fixed a5fc5fc623349498dc4ca673c18408e4ae73d251.